tests/atomizer.js

```diff
@@ -764,6 +764,56 @@ describe('Atomizer()', function () {
             var result = atomizer.getCss(config);
             expect(result).to.equal(expected);
         });
+        it('returns expected css value declared in custom with variable substitution', function () {
+            var atomizer = new Atomizer();
+            var config = {
+                custom: {
+                    '$main-color': '#000000',
+                    'brand-color': '#400090',
+                    'my-gradient': 'linear-gradient(to bottom, #{$main-color}, #{brand-color})'
+                },
+                classNames: ['Bg(my-gradient)']
+            };
+            var expected = [
+                '.Bg\\(my-gradient\\) {',
+                '  background: linear-gradient(to bottom, #000000, #400090);',
+                '}\n'
+            ].join('\n');
+            var result = atomizer.getCss(config);
+            expect(result).to.equal(expected);
+        });
+        it('returns expected css value declared in custom with nested variable substitution', function () {
+            var atomizer = new Atomizer();
+            var config = {
+                custom: {
+                    'black': '#000000',
+                    '$main-color': '#{black}',
+                    'brand-color': '#400090',
+                    'my-gradient': 'linear-gradient(to bottom, #{$main-color}, #{brand-color})'
+                },
+                classNames: ['Bg(my-gradient)']
+            };
+            var expected = [
+                '.Bg\\(my-gradient\\) {',
+                '  background: linear-gradient(to bottom, #000000, #400090);',
+                '}\n'
+            ].join('\n');
+            var result = atomizer.getCss(config);
+            expect(result).to.equal(expected);
+        });
+        it('avoids infinite loops in custom with nested variable substitution', function () {
```

Comment:
Should we add a test for when we go back 10?

Reply:
Current test uses a looping reference to reach 10

Reply:
Also, I've increased the limit to 20, just in case.